modules/core/src/main/scala/lucuma/odb/api/repo/TopLevelRepo.scala

```diff
@@ -12,6 +12,7 @@ import cats.data._
 import cats.effect.concurrent.Ref
 import cats.syntax.apply._
 import cats.syntax.applicative._
+import cats.syntax.either._
```

Comment:
Any reason not to just import `cats.implicits._`?

Reply:
Not really.  I just had a vague sense that a la carte imports are preferred.  What is the consensus?